packages/yoga/src/Result/native/Result/TextWithBadge/styles.ts

```diff
@@ -0,0 +1,42 @@
+import React, { ReactNode } from 'react';
+import styled, { css } from 'styled-components';
+import Text from '../../../../Text';
+import Box from '../../../../Box';
+
+export const StyledBoxContainer = styled(Box)<{
+  containerWidth?: number | null;
+  children: ReactNode;
+}>`
+  ${({ containerWidth }) =>
+    css`
+      flex-direction: row;
+      align-items: center;
+      justify-content: flex-end;
+      position: relative;
+      width: ${containerWidth}px;
+    `}
+`;
+
+export const StyledText = styled(Text.Body1)<{
+  onLayout: (event: any) => void;
```

Comment:
Yo @ffeliperocha!

In our initiative to add Typescript to our project, we would also like to avoid `any` types as much as possible.

I want to ask you to add a type for `onLayout`. It would look nice 🚀